@uportal/esco-content-menu-lit/README.md

```diff
@@ -0,0 +1,643 @@
+
+```xml
+<dependency>
+    <groupId>org.webjars.npm</groupId>
+    <artifactId>uportal__esco-content-menu-lit</artifactId>
+    <version>{version number goes here}</version>
+</dependency>
+```
+
+_install with gradle_
+
+```gradle
+compile 'org.webjars.npm:uportal__esco-content-menu-lit:{version number goes here}'
+```
+
+## Usage as Web Component
+
+### internationalization
+
+The `hamburger-menu`, `content-grid`, and `content-menu` components support an internationalization mixin. This adds the additional prop `messages` which can be applied to override the default text messages in the various components. This property will trickle down to the PortletCard and ActionFavorites components, as well.
+
+For example:
+
+```html
+<esco-hamburger-menu
+  messages='[{"locales": ["en", "en-US"], "messages": { "message": {"favorites": { "add": "Add me to your favorites!" } }}}]'
+>
+</esco-hamburger-menu>
+```
+
+The available messages to override are as follows:
+
+```json
+{
+  "message": {
+    "services": {
+      "title": "All services",
+      "filter": "Find a service..."
+    },
+    "favorites": {
+      "add": "Add to favorites",
+      "remove": "Remove from favorites",
+      "title": "My Favorites",
+      "empty": "No favorite defined"
+    },
+    "filter": {
+      "selectOption": "All categories"
+    },
+    "userInfoPortletUrl": {
+      "title": "See my account informations"
+    },
+    "userChangeEtabUrl": {
+      "title": "Select an other organization"
+    },
+    "buttons": {
+      "logout": "Sign out",
+      "menuClose": "Close menu"
+    }
+  }
+}
+```
+
+### The hamburger menu
+
+This is the main component that show a hamburger menu and that open an entire page with `content-menu` component.
+
+#### Html
+
+```html
+<script
+  src="/resource-server/webjars/uportal__esco-content-menu/dist/js/esco-content-menu.min.js"
+  defer
+></script>
+
+<esco-hamburger-menu
+  default-org-logo="..."
+  api-url-org-info="..."
+  portlet-api-url="..."
+  layout-api-url="..."
+  organization-api-url="..."
+  user-info-api-url="..."
+  context-api-url="/uPortal"
+  sign-out-url="/uPortal/Logout"
+  default-org-logo="https://www.toureiffel.paris/sites/default/files/styles/1440x810/public/2017-10/monument-landing-header-bg_0.jpg?itok=_dSLLBlZ"
+  user-info-portlet-url="/uPortal/user-details"
+  favorites-portlet-card-size="small"
+  grid-portlet-card-size="auto"
+  hide-action-mode="never"
+>
+</esco-hamburger-menu>
+```
+
+For some integration you could need a bit more, like into uPortal you will need to add a parent div and to apply on his closest parent `section` a style `float: left`.
+
+#### Properties
+
+- `messages`: type: `Array of Objects`, see [internationalization](#internationalization)
+- `context-api-url`: type: `String`, default: `/uPortal`, usefull to provide a different uPortal context on which to do request
+- `favorite-api-url`: type: `String`, default: `/uPortal/api/layout`, the uri/url of the favorites api
```

Comment:
Nice that this is separated out - `/api/portlet-list` is in the works!

Reply:
@cbeach47 Yes but there is a breaking change, the ordering isn't managed by code and I think that the json format isn't the same...